crl/validator_test.go

```diff
@@ -142,12 +146,27 @@ func TestValidator_IsRevoked(t *testing.T) {
 		assert.True(t, crlValidator.IsSynced(0))
 	})
 
-	t.Run("should return false if the certificate was not revoked even though the bit was set", func(t *testing.T) {
-		t.Parallel()
+	t.Run("should return false if the crl is expired", func(t *testing.T) {
+		oldNowFunc := nowFunc
```

Comment:
I think you're referring to the inline now func within the function now, due to shadowing. So give it an alternative name.

And restore the `nowFunc` in a `defer` or `t.Cleanup()` to make sure it's executed if the function exits prematurely.